.github/workflows/os_x_staticbuild.yml

```diff
@@ -10,16 +10,16 @@ jobs:
         uses: actions/checkout@v2
       - name: Install Dependencies
         run: |
-          brew install nasm automake ninja libtool cmake pkgconfig protobuf
+          brew install nasm automake ninja libtool cmake pkgconfig protobuf hdf5 zlib
+          python3 -m pip install --user -r ci/docker/install/requirements
```

Comment:
I suggest not to use `ci/docker/install/requirements` here but list the dependencies explicitly. Most of the things in `ci/docker/install/requirements` are not needed for this test and having a single requirements file used for a that is used from an opaque number of locations for an variety of purposes makes it very hard to maintain that file.